postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -465,6 +547,29 @@ def _explain_pg_stat_activity(self, rows):
                     tags=self._tags + ["error:collect-plan-for-statement-crash"] + self._check._get_debug_tags(),
                     hostname=self._check.resolved_hostname,
                 )
+        return events, self._create_activity_event(active_session_rows)
+
+    def _create_activity_event(self, active_query_rows):
+        # Only send an event if we are configured to do so, and
+        # don't report more often than the configured collection interval
+        elapsed_s = time.time() - self._time_since_last_activity_event
+        if elapsed_s < self._activity_coll_interval or not self._activity_coll_enabled:
+            return None
+        # run query to get number of connections by app/user
+        active_connections = self._get_active_connections()
+        self._time_since_last_activity_event = time.time()
+        event = {
+            "host": self._db_hostname,
+            "ddsource": "postgres",
+            "dbm_type": "activity",
+            "collection_interval": self._activity_coll_interval,
+            "ddagentversion": datadog_agent.get_version(),
+            "ddtags": self._tags_no_db,
+            "timestamp": time.time() * 1000,
+            "postgres_activity": active_query_rows,
+            "connections": active_connections,
```

Comment:
Should this be `postgres_connections`?

Reply:
at first i thought about trying to make what we return here generic for each dbms type, but that doesn't seem worth it. updated in this commit https://github.com/DataDog/integrations-core/pull/9866/commits/fdbf42c7e50cdc03332776e58bced9f1cd08c9fd